src/styles/AuthForm.module.scss

```diff
@@ -0,0 +1,61 @@
+@import "colors";
+
+.form {
+  width: 100%;
+  text-align: center;
+  max-height: 18rem;
+  overflow-y: auto;
+  margin-top: 0.5rem;
+
+  & > div:first-child input {
```

Comment:
so once you have something like this, such a specific query, take a step back. Is there a change to make it simple in a way to lets say add a class name to the input?